### PR TITLE
conditionally create award object

### PR DESCRIPTION
**Pull request by fabianvf**
so that validation errors don't occur due to missing conditionally required components

### CODE REVIEW

File: scrapi/harvesters/crossref.py [hunk 1]
```diff
@@ -37,18 +37,21 @@ def process_contributor(author, orcid):
 
 def process_sponsorships(funder):
     sponsorships = []
+
     for element in funder:
-        sponsorship = {
-            'sponsor': {
+        sponsorship = {}
+
+        if element['name']:
```
Comment (erinspace) [inline scrapi/harvesters/crossref.py]: should this be element.get() to avoid a key error? Same with line 49?

**Comment by erinspace** [comment]
Looks good! This might be a different PR, but maybe as we're changing crossref now it could be added as a quick fix: we're not capturing DOI in the objectUris field, nor is the canonicalUri field repeated in the objectUris.

**Comment by fabianvf** [comment]
True, I'll leave that for a different PR though. We can do an audit of existing harvesters at the beginning of this epic, maybe :)

**Comment by erinspace** [comment]
Ok cool!

**Pull request merged**